core/src/labels/labelMesh.cpp

```diff
@@ -1,8 +1,14 @@
 #include "labels/labelMesh.h"
 #include "labels/label.h"
+#include "shaderProgram.h"
 
 namespace Tangram {
 
+GLuint s_quadIndexBuffer = 0;
+int s_quadGeneration = -1;
```

Comment:
Little nitpick but can we declare those as static so the use `s_*` would make sense here.